<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>798</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="twComics">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbDatasets">
     <property name="title">
      <string>Datensatz</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>18</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bewertung:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Größe:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="sbRating">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="specialValueText">
         <string>Punkte eintragen</string>
        </property>
        <property name="suffix">
         <string> Punkte</string>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>-100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="sbSize">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="specialValueText">
         <string>Größe eintragen</string>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="editName">
        <property name="placeholderText">
         <string>Name eintragen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Aktion</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Datensatz hinzufügen</string>
        </property>
        <property name="icon">
         <iconset resource="comicdb.qrc">
          <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnChange">
        <property name="text">
         <string>Datensatz ändern</string>
        </property>
        <property name="icon">
         <iconset resource="comicdb.qrc">
          <normaloff>:/icons/change.png</normaloff>:/icons/change.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Datensatz löschen</string>
        </property>
        <property name="icon">
         <iconset resource="comicdb.qrc">
          <normaloff>:/icons/del.png</normaloff>:/icons/del.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Sprachauswahl</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbDe">
        <property name="text">
         <string notr="true">Deutsch</string>
        </property>
        <property name="icon">
         <iconset resource="comicdb.qrc">
          <normaloff>:/icons/flags/flag_de.png</normaloff>:/icons/flags/flag_de.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbEn">
        <property name="text">
         <string notr="true">English</string>
        </property>
        <property name="icon">
         <iconset resource="comicdb.qrc">
          <normaloff>:/icons/flags/flag_en_us.png</normaloff>:/icons/flags/flag_en_us.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>twComics</tabstop>
  <tabstop>editName</tabstop>
  <tabstop>sbRating</tabstop>
  <tabstop>sbSize</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnChange</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>rbDe</tabstop>
  <tabstop>rbEn</tabstop>
 </tabstops>
 <resources>
  <include location="comicdb.qrc"/>
 </resources>
 <connections/>
</ui>
